flight/tasks/hover_task.py

```diff
@@ -3,7 +3,7 @@
 from task_base import TaskBase
 from .. import constants as c
 
-KP = 1
```

Comment:
Again, these constants should be documented.

Reply:
I fixed this issue in the `feature/imports` pull request.